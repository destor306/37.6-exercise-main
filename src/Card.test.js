import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

//caption, src, currNum, totalNum
it('should render', ()=>{
    render(<Card caption="Test Card" src="test.com" currNum={1} totalNum={3}/>)

})

it('should match snapshot', ()=>{
    const {asFragment} = render(<Card caption="Test Card" src="test.com" currNum={1} totalNum={3}/>)
    expect(asFragment()).toMatchSnapshot();
})