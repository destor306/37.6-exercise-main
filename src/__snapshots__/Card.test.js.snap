// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test Card
    </h4>
    <img
      alt="Test Card"
      class="Card-image"
      src="test.com"
    />
    <small
      class="Card-small"
    >
      Image 1 of 3.
    </small>
  </div>
</DocumentFragment>
`;
